src/inspect_ai/agent/_human/install.py

```diff
@@ -201,25 +203,36 @@ def human_agent_install_sh() -> str:
     # copy command script
     cp {TASK_PY} $HUMAN_AGENT
 
-    # append to .bashrc
-    cat {BASHRC} >> ~/{BASHRC}
+    # get user's home directory
+    USER="{user or ""}"
+    if [ -z "$USER" ]; then
+        USER=$(whoami)
+    fi
+    USER_HOME=$(getent passwd $USER | cut -d: -f6)
+
+    # append to user's .bashrc
+    cat {BASHRC} >> $USER_HOME/{BASHRC}
```

Comment:
This code runs as the sandbox environment's default user, so we can't assume that `~` is `user`'s home directory. Instead we use `getent` to look up the user's home directory.